src/LocalTime.js

```diff
@@ -427,6 +427,7 @@ export class LocalTime extends Temporal /** implements Temporal, TemporalAdjuste
      * @private
      */
     _get0(field) {
+        const ham = MathUtil.intMod(this._hour, 12); 
```

Comment:
this calculation is unnecessary for all cases except CLOCK_HOUR_OF_AMPM, please only do the calculation inside the corresponding case